x/twasm/contract/incoming_msgs.go

```diff
@@ -196,6 +196,40 @@ func (p *GovProposal) UnmarshalJSON(b []byte) error {
 			}
 			return nil
 		},
+		"migrate_contract": func(b []byte) error {
+			var proxy = struct { // todo: better use wasmvmtypes.MigrateMsg when names match
+				Contract string `json:"contract"`
+				CodeID   uint64 `json:"code_id"`
+				Msg      []byte `json:"msg"`
```

Comment:
This is still called `migrate_msg` in the main branch. Better stick to it. Or do you want to include that change in the new (v0.6.3) poe-contracts?

Reply:
The mapping was broken before as the msg field from the contract was encoded. This fixes the issue on the tgrade side. It is not used currently by PoE (main) but I prefer to have this fixed after all this debugging.

Reply:
In PoE main the wasm message is used, which has a different name for the `msg` field. This is not related to this fix.